cmd/bnsapi/main.go

```diff
@@ -68,13 +68,17 @@ func run(conf Configuration) error {
 	rt := http.NewServeMux()
 	rt.Handle("/info", &handlers.InfoHandler{})
 	rt.Handle("/blocks/", &handlers.BlocksHandler{Bns: bnscli})
+	rt.Handle("/account/nonce/address/", &handlers.NonceAddressHandler{Bns: bnscli})
+	rt.Handle("/account/nonce/pubkey/", &handlers.NoncePubKeyHandler{Bns: bnscli})
 	rt.Handle("/username/owner/", &usernameHandlers.OwnerHandler{Bns: bnscli})
+	rt.Handle("/username/resolve/", &usernameHandlers.OwnerHandler{Bns: bnscli})
```

Comment:
`/username/resolve/thematrix*iov` doesn't work for me when I run `TENDERMINT=https://rpc.cluster-mainnet.iov.one:443 HTTP=0.0.0.0:8001 bnsapi`.  I get
```
dev username_handler.go:27: /username/resolve/thematrix*iov
dev username_handler.go:28: thematrix*iov
dev username_handler.go:31: cannot decode hex: encoding/hex: invalid byte: U+0074 't' [iov-one/weave@v1.0.0/errors/errors.go:228]
```